assets/app/widgets/suse-modal/suse-modal.directive.jade

```diff
@@ -0,0 +1,9 @@
+.modal-content
+    .modal-header
+        button.close(type='button', ng-click='dismiss()', aria-label='Close')
+            span(aria-hidden='true') ×
+        h4#suseModalLabel.modal-title {{ error.title }}
```

Comment:
I would rather call this something like `suse-error-modal` as this is the current use case.